File: lib/plane_spec.rb
require 'plane'

describe 'Planes can land in the airport', :focus => true do
  #expect airport to accept land plane and return plane/true
  it 'responds to #land_plane' do
    new_plane = Plane.new
    expect(subject.land_plane(new_plane)).to eq true
  end
end

File: jasmine/spec/planeSpec.js
'use strict';

describe('Plane',function(){
  var plane;
  var airport;
  beforeEach(function(){
    plane = new Plane();
    airport = jasmine.createSpyObj('airport',['clearForLanding','clearForTakeOff']);
  });
  it('can land at an airport', function(){
    plane.land(airport);
    expect(airport.clearForLanding).toHaveBeenCalledWith(plane);
  });
  it('can takeoff from an airport', function(){
    plane.land(airport);
    plane.takeoff();
    expect(airport.clearForTakeOff).toHaveBeenCalled();
  });
});
